src/utils.ts

```diff
@@ -33,6 +33,7 @@ const appendScript = (scriptText: string, scriptId: string): boolean => {
     const existingScript = document.getElementById(scriptId) as HTMLScriptElement;
     const script = existingScript || document.createElement('script');
     script.id = scriptId;
+    script.nonce = scriptId;
```

Comment:
I think we'll want to allow the `nonce` to be passed in by the consumer since they are responsible for generating it. Right now, it looks like this `nonce` will always be `hotjar-init-script`.

Reply:
Ok, I fixed it

Reply:
OK, I fixed